doc/src/advanced_setup.md

```diff
@@ -0,0 +1,31 @@
+When you setup LALRPOP, you create a `build.rs` file that looks something
+like this:
+
+```rust
+extern crate lalrpop;
+
+fn main() {
+    lalrpop::process_root().unwrap();
+}
+```
+
+This `process_root()` call simply applies the default configuration:
+so it will transform `.lalrpop` files into `.rs` files *in-place* (in
+your `src` directory), and it will only do so if the `.lalrpop` file
+has actually changed. But you can also use the
+[`Configuration`][config] struct to get more detailed control.
+
+[config]: https://github.com/lalrpop/lalrpop/blob/master/lalrpop/src/api/mod.rs
```

Comment:
Maybe we should publish the rustdocs too? Linking to docs.rs requires specifying a version, from what I could tell.

Reply:
You can link to https://docs.rs/lalrpop/ but that just sends you to the crate docs. Linking below that requires a specific version.

Reply:
https://docs.rs/lalrpop/*/lalrpop/ will take you to the newest version of lalrpops docs (the "version" is actually understood as semver)

Reply:
Ah, nice.